src/request_list.js

```diff
@@ -59,7 +60,11 @@ const ensureUniqueKeyValid = (uniqueKey) => {
  * ```
  *
  * @param {Object} options
- * @param {Array} options.sources Function that processes a request. It must return a promise.
+ * @param {Array} options.sources An array of sources for the `RequestList`. Its contents can either be just plain objects,
+ *                                defining at least a 'url' property, or instances of the `Request` class. See the `Request`
```

Comment:
"defining at least the 'url' property"
please make the "Request" link to its documentation, otherwise it's difficult for users to find it